DataFormats/Detectors/PHOS/include/DataFormatsPHOS/EventData.h

```diff
@@ -37,6 +39,7 @@ struct EventData {
     mClusters = gsl::span<const Cluster>();
     mCells = gsl::span<const InputType>();
     mCellIndices = gsl::span<const int>();
+    mMCCellLabels = std::vector<gsl::span<const o2::phos::MCLabel>>();
   }
 
   ClassDefNV(EventData, 1);
```

Comment:
Since we added new data member std::vector<gsl::span...>, the ClassDefNV should be incremented.